app/javascript/styles/_resultsPanel.scss

```diff
@@ -83,15 +83,16 @@
     word-wrap: break-word;
   }
   .cell-count {
-    background-color: #74ae43;
+    color: #000000;
+    background-color: #deeed0;
     font-weight: normal;
-    margin-right: 0.5em
+    margin-right: 0.5em;
   }
   .facet-match, .gene-match {
     background-color: #fff;
     margin-right: 0.5em;
-    color: #74ae43;
-    border: 1px solid #74ae43;
+    color: #4C841F;
+    border: 1.25px solid #59a319;
```

Comment:
Indeed, the border change was more so because I thought it looked nicer with the new colors not that it was needed for accessibility. However if others don't agree I can change it back

Reply:
I defer to your preference here!